custom/enikshay/tasks.py

```diff
@@ -0,0 +1,109 @@
+import datetime
+from collections import defaultdict
+from django.utils.dateparse import parse_datetime, parse_date
+
+from .case_utils import get_adherence_cases_between_dates_from_episode
+from corehq.apps.fixtures.models import FixtureDataItem
+
+
+DOMAIN = 'enikshay'
```

Comment:
Yeah. I want to look at flexibility of this. Just wanted to get spec sign off for now